### ISSUE TITLE
Welch Labs — Neural Networks Demystified

### ISSUE THREAD

**Issue by atticusrussell** [atticusrussell/ballbot#207]
**Neural Networks Demystified** — Welch Labs. Neural network basics with the implementation + calculus, building a net in Python.

[Full playlist](https://www.youtube.com/playlist?list=PLiaHhY2iBX9hdHaRr6b7XevZtgZRa1PoU)

- [ ] [Part 1 — Data and Architecture](https://youtu.be/bxe2T-V8XRs)
- [ ] [Part 2 — Forward Propagation](https://youtu.be/UJwK6jAStmg)
- [ ] [Part 3 — Gradient Descent](https://youtu.be/5u0jaA3qAGk)
- [ ] [Part 4 — Backpropagation](https://youtu.be/GlcnxUlrtek)
- [ ] [Part 5 — Numerical Gradient Checking](https://youtu.be/pHMzNW8Agq4)
- [ ] [Part 6 — Training](https://youtu.be/9KM9Td6RVgQ)
- [ ] [Part 7 — Overfitting, Testing, and Regularization](https://youtu.be/S4ZUwgesjS8)
